.teamcity/patches/vcsRoots/EdFiOds.kts

```diff
@@ -0,0 +1,41 @@
+package patches.vcsRoots
```

Comment:
add header please

Reply:
This file was accidentally added by Travis and will be removed.